R/reportEmi.R

```diff
@@ -1155,6 +1375,47 @@ reportEmi <- function(gdx, output = NULL, regionSubsetList = NULL, t = c(seq(200
                           "Carbon Management|Carbon Capture (Mt CO2/yr)")
   )
 
+  # add materials-related carbon management variables only if available
+  if (!is.null(vm_plasticsCarbon)){
+  # carbon management: flows of carbon going into materials
+  out <- mbind(out,
+
+               # carbon in plastics
+               setNames(dimSums(vm_plasticsCarbon, dim = 3) * GtC_2_MtCO2,
+                        "Carbon Management|Materials|+|Plastics (Mt CO2/yr)"),
+               # carbon in carbon fibres
+               #setNames(dimSums( carbonfibersVariable * GtC_2_MtCO2,
+               #        "Carbon Management|Materials|+|Carbon fibers (Mt CO2/yr)"),
+               # total co2 in materials
+               # other materials need to be added here
+               setNames(dimSums(vm_plasticsCarbon, dim = 3) * GtC_2_MtCO2,
+                        "Carbon Management|Materials (Mt CO2/yr)")
+  )
+
+  # carbon management: fate of plastics
+  out <- mbind(out,
+
+            # carbon in plastics that are incinerated
+            setNames(dimSums(vm_incinerationEmi, dim = 3) * GtC_2_MtCO2,
+                    "Carbon Management|Materials|Plastics|++|Incineration (Mt CO2/yr)"),
+            # carbon in plastics with other fate
+            setNames(dimSums(vm_nonIncineratedPlastics, dim = 3) * GtC_2_MtCO2,
```

Comment:
What about ``vm_feedstockEmiUnknownFate`` here? Are they part of ``vm_nonIncineratedPlastics``? Or are they not accounted as CO2 going into materials?